image/gimp_brush.ksy

```diff
@@ -0,0 +1,32 @@
+meta:
+  id: gimp_brush
+  title: GIMP (GNU Image Manipulation Program) brush version 2 file
+  file-extension: gbr
+  license: CC0-1.0
```

Comment:
This may (or may not, if you have clean room or black box reverse engineered it) violate the terms of GPL, under which that repo is licensed.

Reply:
This was derived from a documentation file released by the GIMP developers themselves. I am pretty confident that this would not constitute a license violation. I will verify with the GIMP team.